### ISSUE TITLE
parameters

### ISSUE THREAD

**Issue by mab85** [sibofeng/PILAE#1]
@sibofeng How you adjust parameters?

**Comment by sibofeng** [comment]
I update the code, hope can help you to understand the mean of this algorithm. If you want to know more about this algorithm, you can read the paper link under the README and its reference.

**Comment by mab85** [comment]
@sibofeng Thanks for reply but i ask are you implement PIL and PILAE in same file?.

**Comment by sibofeng** [comment]
Yes, more specifically, PIL is an optimization algorithm. In this work, PIL algorithm used to train autoencoder for feature learning and to train MLP for classification.

**Comment by mab85** [comment]
@xiaobotianxie Can you describe in more details how to run the project for own dataset (use English)?
